test/ifroute_test.rb

```diff
@@ -81,6 +81,8 @@
 end
 
 describe "Routing#write_routes" do
+  Yast.import "FileUtils"
```

Comment:
this should not be needed as it is already imported in Routing or I overlook something?